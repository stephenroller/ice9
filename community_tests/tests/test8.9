# write 10/3;

write 10 / 3;
