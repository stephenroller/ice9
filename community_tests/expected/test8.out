Loading file: generated/test8.tm
3 
Number of instructions executed = 16
